docs/pages/blog/matheus-wichman-joining.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Matheus Wichman joins Material-UI
+description: We are excited to share that Matheus Wichman has joined Material-UI.
+date: 2021-04-05T00:00:00.000Z
+authors: ['oliviertassinari']
+card: true
+---
+
+We are excited to share that [Matheus Wichman](https://github.com/m4theushw) has joined Material-UI.
+He has started a couple of days ago full-time and is now part of the advanced components team.
+
+Before joining Material-UI, Matheus worked as a software engineer at a consulting agency in Brazil.
+He has spent the last three years focusing on React and the frontend world.
+His last project was for a financial institution.
```

Comment:
```suggestion
his last project was for a financial institution.
```

"his last project was for a financial institution, where he [fill in the blank].
```
"

Reply:
@oliviertassinari I can update this one

Reply:
@m4theushw Please do 👍